ferrocene/ci/scripts/github_app_auth.py

```diff
@@ -54,6 +46,23 @@ def req(session, method, url):
 
     token = req(http, "POST", installations[0]["access_tokens_url"])["token"]
 
+    # We never want to print this on CI.
+    if 'CI' in os.environ and os.environ['CI'] == True:
+        print(f"::add-mask::{token}")
+    
+    return token
+
+if __name__ == "__main__":
```

Comment:
These changes are unnecessary and not strictly related to this PR, but act as general cleaning as I was previously using them in an earlier revision.

I think it's worth including this despite being unrelated.